internal/maven/repository.go

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright 2022.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package maven
+
+import "strings"
+
+const (
+	mavenCentralUrl = "https://repo1.maven.org/maven2"
+)
+
+type Repository struct {
+	url string
+}
+
+func MavenCentral() Repository {
```

Comment:
Perhaps just `var MavenCentral = Repository{mavenCentralUrl}`?

Will there ever be `Repository` state for MavenCentral that you don't want globally shared?

Otherwise, just expose the `mavenCentralUrl` so it's easy to do this yourself. It's a bit odd to hard-code a specific service in your API in this way.